### ISSUE TITLE
GenderOther Variable Compromised

### ISSUE THREAD

**Issue by lama-ahmad** [freeCodeCamp/2017-new-coder-survey#9]
There seems to be a problem with the GenderOther variable. When I loaded the dataset in R, I wanted to convert the GenderOther variable to a factor and got this:

`51 Levels: 54 a abcdefghijklmnopqrstuvwxyz apache attack helicopter Apache attack helicopter ... why the hell are here more than two genders?!`

**Comment by erictleung** [comment]
Yes, unfortunately, this has some irrelevant values. This was not ideal, but I didn't have time to flush out these values. I'm currently working on the 2018 dataset and there appears to be the same issue.

If you happen to write some code to help clean this part up, feel free to share it here for others to use as well until there is fix on this data. Thanks!
